[DataGridPro] Expose the field of the tree data grouping column as a constant (#3549)
mui-org/material-ui-x: docs/src/pages/components/data-grid/group-pivot/group-pivot.md

{{"demo": "pages/components/data-grid/group-pivot/CustomGroupingColumnTreeData.js", "bg": "inline", "defaultCodeOpen": false}}

#### Accessing the grouping column field

> If you want to access the grouping column field, for instance to pin it, you can use the `GRID_TREE_DATA_GROUPING_FIELD` constant.

Reviewer: Explicitly writing "column pinning" makes easier to find with <kbd>CTRL</kbd> + <kbd>f</kbd>. 

```suggestion
If you want to access the grouping column field, for instance, to use it with column pinning, the `GRID_TREE_DATA_GROUPING_FIELD` constant is available.
```

```tsx

> <DataGrid

Reviewer: ```suggestion
<DataGridPro
```
